test/forces.jl

```diff
@@ -37,7 +39,12 @@ end
         basis = PlaneWaveBasis(model, Ecut, kgrid=[2, 1, 2])
 
         n_bands = 10
-        scfres = self_consistent_field(basis, n_bands=n_bands, tol=1e-12, diagtol=1e-12)
+        is_converged = scf_convergence_density_difference(5e-11)
+        determine_diagtol = scf_determine_diagtol(ratio_ρdiff=0.01)
```

Comment:
is this really needed? I'd rather avoid tweaking the diagtol in the tests (which is a nice thing with this PR). The is_converged should be sufficient. Feel free to take bigger tolerances for the finite difference check if needed

Reply:
Playing with the FD parameter helps. I used that to get it a bit more stable.